env/Tensorflow_probability.yml

```diff
@@ -0,0 +1,33 @@
+name: Tensorflow
+channels:
+  - conda-main
+  - conda-mo-internal
+  - conda-forge
+  - conda-r
+dependencies:
+  - python=3.7
+  - tensorflow-probability=0.12
+  - ipykernel
+  - tensorboard=2.4
+  - notebook
+  - matplotlib
+  - keras-applications
+  - pydot
+  - graphviz
+  - cf_units
+  - cf-units=3.0.0
+  - iris=3.1.0
+  - dask
+  - seaborn
+  - scikit-learn
```

Comment:
I think it would be good to add at least major and minor version numbers to these, other it might become quite slow to solve. sojust dask=x.y , muc like you've done with iris and tensorboard.

Reply:
I intentionally left this quite open so conda could do most of the work when figuring out package versions, so if some of the specified package versions go missing this could still work. I including the lock file which this env could be created from which should be must faster to resolve, but would fail if said packages went missing. 

Happy to make it more explicit though - thoughts?

Reply:
there's no "right answer" here of course. I've just found that lately that very quickly the completely open env defs quickly fall over. Being a bit more pinned down is useful in being more confident it will work. Its still more portable than a lock file, but slightly more robust.

Reply:
especially on the met office system, where it can takes ages to resolve the environment if there are a lot of versions to resolve.